### PR TITLE
label for deployment ory-oathkeeper with app: oathkeeper

### PR DESCRIPTION
**Pull request by lauraanddola**
to fix the `source_app=unknown` issue on ory-oathkeeper although it is a part of the Istio mesh  (see [here](https://github.tools.sap/kyma/backlog/issues/809#issuecomment-141071))

### CODE REVIEW

File: resources/ory/charts/oathkeeper/templates/_helpers.tpl [hunk 1]
```diff
@@ -35,6 +35,7 @@ Create chart name and version as used by the chart label.
 Common labels
 */}}
 {{- define "oathkeeper.labels" -}}
+app: {{ .Chart.Name }}
```
Comment (ebensom) [inline resources/ory/charts/oathkeeper/templates/_helpers.tpl:38]: This should be `{ include "oathkeeper.name" . }}`

Comment (lauraanddola) [reply resources/ory/charts/oathkeeper/templates/_helpers.tpl]: get it and I will change it.

**Review by piotrmsc — APPROVED** [review]

**Review by Tomasz-Smelcerz-SAP — APPROVED** [review]

**Pull request merged**